src/annotator/test/guest-test.js

```diff
@@ -605,13 +572,18 @@ describe('Guest', () => {
 
   describe('#createAnnotation', () => {
     it('adds metadata to the annotation object', () => {
+      class FakeDocumentMeta {
+        get metadata() {
+          return { title: 'hello' };
+        }
+
+        uri() {
+          return 'http://example.com';
+        }
+      }
+      $imports.$mock({ './plugin/document': FakeDocumentMeta });
+
       const guest = createGuest();
-      sinon.stub(guest, 'getDocumentInfo').returns(
-        Promise.resolve({
-          metadata: { title: 'hello' },
-          uri: 'http://example.com/',
-        })
-      );
```

Comment:
This changes the mocking of document metadata extraction in these tests to use our normal approach rather than monkey-patching a method of the class being tested.